network-store-server/src/main/java/com/powsybl/network/store/server/migration/MigrationController.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.server.migration;
+
+import com.powsybl.network.store.model.*;
+import com.powsybl.network.store.server.NetworkStoreRepository;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+@RestController
+@RequestMapping(value = "/" + NetworkStoreApi.VERSION + "/migration")
+@Tag(name = "Network store migration")
+public class MigrationController {
+
+    @Autowired
+    private NetworkStoreRepository repository;
+
+    @PutMapping(value = "/{networkId}")
+    @Operation(summary = "Migrate limits of a network")
+    @ApiResponses(@ApiResponse(responseCode = "201", description = "Successfully migrated limits from V2.11.0 to new model"))
```

Comment:
Typo 201? in the code it's ResponseEntity.ok() which is 200 ?

Reply:
done